build/yamls/flow-aggregator.yml

```diff
@@ -203,6 +203,21 @@ data:
 
       # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
       #tlsMinVersion:
+    
+    # clickHouse contains ClickHouse related configuration options.
+    #clickHouse:
+      # Username of ClickHouse client.
+      #username: "clickhouse_operator"
+      # Password of ClickHouse client.
+      #password: "clickhouse_operator_password"
+      # DB name containing antrea table.
+      #database: "default"
+      # URL to ClickHouse DB.
+      #dbUrl: "tcp://<click-house-svc>:9000"
```

Comment:
Just a curiosity: can we expose clickhouse as a k8s svc? Unless I did not search correctly I cannot see it in flow-visibility.yaml

Reply:
the service is created by clickhouse operator. latest change defaults this address via cluster service